packages/components/src/components/functional/CurrentAccountsList/CurrentAccountsList.tsx

```diff
@@ -67,6 +67,10 @@ const CurrentAccountsMain = ({
   const [banks, setBanks] = useState<BankSettings[] | undefined>(undefined)
   const { userSettings, updateUserSettings } = useUserSettings()
 
+  const [systemErrorTitle, setSystemErrorTitle] = useState<string | undefined>(undefined)
+  const [systemErrorMessage, setSystemErrorMessage] = useState<string | undefined>(undefined)
```

Comment:
Just a small suggestion - how about combining these two into a single state with an interface specifically for system errors? It would then be something like this:

So it would look like this instead:
```suggestion
  const [systemError, setSystemError] = useState<SystemError | undefined>(undefined)
```

Reply:
Ah yes. Will do that

Reply:
Thanks!